posthog/temporal/batch_exports/backfill_batch_export.py

```diff
@@ -173,7 +173,7 @@ async def backfill_schedule(inputs: BackfillScheduleInputs) -> None:
         if details:
             # If we receive details from a previous run, it means we were restarted for some reason.
             # Let's not double-backfill and instead wait for any outstanding runs.
-            last_activity_details = HeartbeatDetails(*details[0])
+            last_activity_details = HeartbeatDetails(*details)
```

Comment:
`details` is already the tuple recorded by temporal. By doing `details[0]` we access the first item of the tuple (`workflow_id`), which is unpacked into a bunch of strings. This is, clearly, not good.